doc/user/content/overview/arrangements.md

```diff
@@ -0,0 +1,130 @@
+
+### Example arrangements
+
+Let's take a look at some of the arrangements that Materialize creates for different circumstances.
+
+#### `COUNT`
+
+Some of the simplest arrangements are those for Differential Dataflow operators. For example, the `COUNT` operator, which Materialize uses for internal logging, has two arrangements:
+
+* An arrangement for the input, which indexes the key whose values will be counted
+* An arrangement for the output, which maintains the results
+
+`COUNT` reads from the input arrangement to count values, and both reads from the output (to see the previous total) and writes to the output (adding the new updates to the previous total).
+
+![Diagram of arrangements for count](/images/arrangements-count.png "Diagram of arrangements for COUNT")
+
+#### Three-way join
+
+Let's say that you want to review orders by shipdate. The query itself is:
+
+```sql
+SELECT
+    l_orderkey,
+    o_orderdate,
+    o_shippriority,
+    sum(l_extendedprice * (1 - l_discount)) AS revenue
+FROM
+    customer,
+    orders,
+    lineitem
+GROUP BY
+    l_orderkey,
+    o_orderdate,
+    o_shippriority
+ORDER BY
+    revenue desc,
+    o_orderdate;
+```
+
+The query is a three-way join between `customer`, `orders`, and `lineitem`, followed by a reduction. For the purposes of this explanation, we're going to focus on the join and skip the reduction.
+
+When joining the three tables, Materialize builds indexes on `customers` + `orders`  and (`customers` + `orders`) + `lineitem`. It can do this because it separates out the work of the join calculation from the work of indexing information or creating arrangements. Each of the original columns and both of the necessary joins create arrangements that can be reused for other queries.
+
+![Diagram of arrangements for a three-way join](/images/arrangements-3-way-join.png "Diagram of arrangements for a three-way join")
+
+Note that this is a simplification for the sake of an example, focusing on the join and excluding the reduction at the end. In practice, if a join is going into a reduction, Materialize does not produce and maintain an intermediate arrangement as large as the join itself.
+
+You can find a more detailed analysis of the dataflow (and the performance benefits of using Materialize) in our blog post on [Joins in Materialize](https://materialize.com/joins-in-materialize).
+
+## Analyzing arrangements
+
+Materialize provides various tools that allow you to analyze arrangements, although they are post hoc tools best used for debugging, rather than planning tools to be used before creating indexes or views. See [Diagnosing Using SQL](/ops/diagnosing-using-sql/), [Monitoring](/ops/monitoring/), and [`EXPLAIN`](/sql/explain/) for more details.
+
+## Reducing memory usage
+
+### Creating indexes manually
+
+You can't create arrangements directly, but in some cases you can manually create indexes that will help Materialize create more sharable arrangements -- and therefore reduce memory usage.
```

Comment:
You can use `CREATE INDEX`. I'm confused about the language in this section, but it seems the main thing to do is call out that it is a command you can use to manually create indexes.